kickoff/jsonexport.py

```diff
@@ -277,3 +280,54 @@ def json_exports_txt():
     response = make_response(render_template("json_exports.txt", jsonFiles=jsonFiles))
     response.mimetype = "text/plain"
     return response
+
+
+def get_releases_for_json(product):
+    releases = getFilteredReleases(product, ["major", "stability", "esr", "dev"], detailledInfo=True)
+    listReleases = defaultdict()
+    for r in releases:
+        version = r[2]
+        category = "esr" if version.endswith("esr") else r[3]
+        key = product + "-" + version
+        listReleases[key] = {
+            "date": r[1],
+            "version": r[0],
+            "product": product,
+            "category": category,
+            "description": r[4],
+            "is_security_driven": r[5] == True
+        }
+    return listReleases
+
+
+@app.route('/json/firefox.json', methods=['GET'])
```

Comment:
Should you define only one endpoint `'/json/{product}.json'` and then pass the product info to `get_releases_for_json`?